README.md

```diff
@@ -1,59 +1,123 @@
 [![Build Status](https://travis-ci.org/dedis/cothority_template.svg?branch=master)](https://travis-ci.org/dedis/cothority_template)
 
-# Template for a new cothority protocol/service/app
-
-Building on the ONet-library available at 
+Navigation: [DEDIS](https://github.com/dedis/doc/tree/master/README.md) ::
+Cothority Template
+
+# Introduction
+
+The following paragraphs give a short introduction in the onet framework and
+should be your starting point if this is a new project for you.
+
+## Terminology
+
+Onet has many concepts and components which are described below:
+
+- Server members of the Onet are called Nodes. They are started by the
+Cothority app from the cothority-repository.
```

Comment:
"started by the cothority app" sounds confusing because cothority isn't just one app, perhaps you want to say "started by one of the cothority apps"?